std-bits/table/src/main/java/org/enso/table/parsing/IdentityParser.java

```diff
@@ -0,0 +1,28 @@
+package org.enso.table.parsing;
+
+import java.util.List;
+import org.enso.table.data.column.builder.object.Builder;
+import org.enso.table.data.column.builder.object.StringBuilder;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.column.storage.StringStorage;
+import org.enso.table.parsing.problems.ProblemAggregator;
+import org.enso.table.read.WithProblems;
+
+/** A parser that just returns its input. Useful as a fallback. */
+public class IdentityParser extends IncrementalDatatypeParser {
```

Comment:
This feels wrong as it will result in duplicating the `string[]`.

Reply:
It returns the original storage, there is no cloning involved.

Reply:
not true in the current implementation - the TypeInferingParser will call the makeBuilder and then clone each value into the array.

While only an array of pointers will do unneeded work,

Reply:
Oh, right. When I was implementing it originally the approach was different and I didn't notice the change. Fair. Maybe then it requires some further thought.

Reply:
I have to keep this functionality, because the `WhitespaceStrippingParser` relies on that - it needs a builder because it can modify the values.

But thanks to the `fallbackParser` change, if whitespace trimming is off, no copying will occur.